python/lsst/pipe/tasks/characterizeImage.py

```diff
@@ -359,33 +368,43 @@ def runDataRef(self, dataRef, exposure=None, background=None, doUnpersist=True):
 
     @pipeBase.timeMethod
     def run(self, exposure, exposureIdInfo=None, background=None):
-        """!Characterize a science image
+        """Characterize a science image
 
         Peforms the following operations:
         - Iterate the following config.psfIterations times, or once if config.doMeasurePsf false:
-            - detect and measure sources and estimate PSF (see detectMeasureAndEstimatePsf for details)
+        - detect and measure sources and estimate PSF (see detectMeasureAndEstimatePsf for details)
         - interpolate over cosmic rays
         - perform final measurement
 
-        @param[in,out] exposure  exposure to characterize (an lsst.afw.image.ExposureF or similar).
+        Parameters
+        ----------
+        exposure :
+            exposure to characterize (an lsst.afw.image.ExposureF or similar).
             The following changes are made:
             - update or set psf
             - set apCorrMap
             - update detection and cosmic ray mask planes
             - subtract background and interpolate over cosmic rays
-        @param[in] exposureIdInfo  ID info for exposure (an lsst.obs.base.ExposureIdInfo).
+
+        exposureIdInfo :
+            ID info for exposure (an lsst.obs.base.ExposureIdInfo).
             If not provided, returned SourceCatalog IDs will not be globally unique.
-        @param[in,out] background  initial model of background already subtracted from exposure
+        background : `lsst.afw.math.BackgroundList`
+            initial model of background already subtracted from exposure
             (an lsst.afw.math.BackgroundList). May be None if no background has been subtracted,
             which is typical for image characterization.
 
-        @return pipe_base Struct containing these fields, all from the final iteration
-        of detectMeasureAndEstimatePsf:
-        - exposure: characterized exposure; image is repaired by interpolating over cosmic rays,
+        Returns
+        -------
+        dmeRes : `struct`
```

Comment:
This is super close to the right format for a struct, but needs some tweaking (see earlier comment).